lib/tapioca/dsl/compilers/active_support_concern.rb

```diff
@@ -66,19 +66,18 @@ class << self
 
           sig { override.returns(T::Enumerable[Module]) }
           def gather_constants
-            # Find all Modules that are:
```

Comment:
The comments here are superfluous IMO, better to clarify the code itself (e.g. by adding a `has_dependencies?` helper)